src/main/resources/application.yaml

```diff
@@ -6,6 +6,12 @@ management:
     web:
       base-path: /
 
+info:
+   app:
+      name: Feature Toggle API
+      description: This application allows to enable/disable features at runtime without deployment.
+      version: 1.0.0
```

Comment:
`build.gradle` says it's `0.0.1`. Let's stick to one source of this information...